tests/test_utils_multiprocess.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from sretoolbox.utils import multiprocess
+from tests import fixture_function
+
+class TestRunProcessStuff(unittest.TestCase):
+    def test_run_normal(self):
+        rs = multiprocess.run(fixture_function.identity, [42, 43, 44], 1)
+        self.assertEqual(rs, [42, 43, 44])
+
+    def test_run_normal_with_exceptions(self):
+        with self.assertRaises(Exception):
```

Comment:
Can we also raise a `SystemExit` to check if it is caught? See comment above.

Reply:
`sys.exit()` is doing that. see the `sys_exit_func` tester func

Reply:
I see. Should there be a test-case using the `sys_exit_func` fixture?

Reply:
ah got lost in my last code restructuring of the tests. now it is in there again. thanks for highlighting this!